src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -187,7 +188,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 End If
             Else
                 ' Nested types (including types in modules) can be any accessibility, and the default is public
-                If (modifiers And DeclarationModifiers.Private) <> 0 Then
+                If (modifiers And (DeclarationModifiers.Private Or DeclarationModifiers.Protected)) =
```

Comment:
No, I don't think so. The caller already handles that by reporting extra access modifiers.